src/localplayer.cpp

```diff
@@ -279,12 +279,10 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 	// This should always apply, otherwise there are glitches
 	sanity_check(d > pos_max_d);
 
-	// TODO: this shouldn't be hardcoded but transmitted from server
-	float player_stepheight = (touching_ground) ? (BS * 0.6f) : (BS * 0.2f);
-
-#ifdef __ANDROID__
-	player_stepheight += (0.6f * BS);
-#endif
+	// Player object property step height is multiplied by BS in
+	// /src/script/common/c_content.cpp and /src/content_sao.cpp
+	float player_stepheight = (m_cao == 0) ? 0.0 :
+		(touching_ground ? m_cao->getStepHeight() : (0.2f * BS));
```

Comment:
I have intentionally removed the hacky and buggy Android autojump for reasons given above and in #6183 
If you support the rest of this but want the Android autojump change a separate PR i can do that.